css/component_styles/slider.css

```diff
@@ -1,3 +1,7 @@
 @import url('./components.css');
 
-/* -------- slider css -------- */
\ No newline at end of file
+/* -------- slider css -------- */
```

Comment:
try to change the colour of slider and remove the comment part

Reply:
But I want it to be of that colour as the user can change its colour as per need and I don't see any issue with the comment to be removed